### PR TITLE
app-dicts/myspell-en: Fix Hypen File Issue

### PR DESCRIPTION
**Pull request by oniGino**
Bug: https://bugs.gentoo.org/589942

* Fix issue where updated hyphen file from dictionary is not used.

### CODE REVIEW

**Comment by mgorny** [comment]
I'm going to merge this but revert stylistic changes to hyphen. Please don't unnecessarily change the style of ebuilds you don't maintain.

File: dev-libs/hyphen/hyphen-2.8.8-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	econf $(use_enable static-libs static)
+}
+
+src_install() {
+	default
+
+	docinto pdf
+	dodoc doc/*.pdf
+
+	if ! use static-libs; then
```
Comment (mgorny) [inline dev-libs/hyphen/hyphen-2.8.8-r1.ebuild]: Why are you making the removal conditional now?

Comment (oniGino) [reply dev-libs/hyphen/hyphen-2.8.8-r1.ebuild]: that is from some older version of hyphen, i didn't add that, 
i updated my -r1 edit to match the current ebuild

**Comment by mgorny** [comment]
Ah, right but please make it update 2019* ebuild.

**Comment by oniGino** [comment]
> Ah, right but please make it update 2019* ebuild.

Done

**Comment by mgorny** [comment]
Thanks!

**Pull request closed without merging**